lib/Ohmbrewer_Onewire.cpp

```diff
@@ -108,14 +123,17 @@ double Ohmbrewer::Onewire::getReading(){
  */
 void Ohmbrewer::Onewire::displayProbeIds(Ohmbrewer::Screen *screen){
     //header
+    screen->displayHeader();
+
     screen->resetTextSize();
     screen->resetTextColor();
+    screen->println("-------------------");
     // Print the section title
-    screen->print("PROBE ID# : TEMP");
+    screen->println("PROBE ID# : TEMP");
 //    screen->println("");
     screen->printMargin(2);
 
-
+    char msg[100];
```

Comment:
Given that we only print 20 characters per line, this seems like a little much...